lib/dynamoid/document.rb

```diff
@@ -13,7 +13,7 @@ module Document
       self.read_only_attributes = []
       self.base_class = self
 
-      Dynamoid.included_models << self
+      Dynamoid.included_models << self unless Dynamoid.included_models.include? self
```

Comment:
I saw a case where the same class had been added to the Dynamoid.included_models more than once. I couldn't find a way to reproduce it so haven't written a test proof.